students/11th/seunghaoh/js/login.js

```diff
@@ -0,0 +1,40 @@
+// const id = document.getElementsByClassName("inputId")[0];
+// const pw = document.getElementsByClassName("inputPw")[0];
+const inputs = document.querySelectorAll(".inputs");
+const thisIsBtn = document.getElementsByClassName("loginBtn")[0];
+
+// id.addEventListener("keyup", function () {
+//   loginBtnActive();
+// });
+
+// pw.addEventListener("keyup", function () {
+//   loginBtnActive();
+// });
+
+inputs[0].addEventListener("keyup", function () {
+  loginBtnActive();
+});
+
+inputs[1].addEventListener("keyup", function () {
+  loginBtnActive();
+});
+
+function loginBtnActive() {
+  // const idVal = id.value;
+  // const pwVal = pw.value;
+  // if (idVal && pwVal) {
+  //   thisIsBtn.setAttribute("style", "opacity: 1");
+  //   thisIsBtn.disabled = false;
+  // } else {
+  //   thisIsBtn.setAttribute("style", "opacity: 0.4");
+  //   thisIsBtn.disabled = true;
+  // // }
+  // const inputsVal = inputs.value;
+  if (inputs[0].value && inputs[1].value) {
+    thisIsBtn.setAttribute("style", "opacity: 1");
+    thisIsBtn.disabled = false;
+  } else {
+    thisIsBtn.setAttribute("style", "opacity: 0.4");
+    thisIsBtn.disabled = true;
+  }
```

Comment:
```suggestion
  const isValid = inputs[0].value && inputs[1].value
  thisIsBtn.setAttribute("style", `opacity: ${isValid ? "1" : "0.4"}`);
  thisIsBtn.disabled = !isValid;
```
이런식으로 사용해줄 수도 있습니다.
1. 삼항연산자를 사용했습니다 (참고 : https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)
2. 그 자체로 boolean 값을 갖는 성질을 이용할 수 있습니다